protostar/cheatable_starknet/cheaters/expect_events_controller.py

```diff
@@ -85,24 +84,22 @@ def compare_expected_and_actual_results(self):
 def match_events(
     expected_events: list[Event], emitted_events: list[Event]
 ) -> EventMatchingResult:
-    expected_events_queue = deque(expected_events)
     event_matchings: list[EventMatching] = []
     for emitted_event in emitted_events:
-        if len(expected_events_queue) == 0 or not should_accept_event_matching(
-            expected_event=expected_events_queue[0], emitted_event=emitted_event
+        if len(expected_events) == 0 or not should_accept_event_matching(
```

Comment:
`len(expected_events) == 0` is simplifiable to `not expected_events`

Reply:
I chose to use `len` variant, to highlight the existence of `expected_events_queue[0]` in the next condition.